install.sh

```diff
@@ -12,6 +12,10 @@ this_dir="$(dirname -- "$( readlink -f -- "$0"; )")";
 echo "this_dir is $this_dir"
 pushd "$this_dir" > /dev/null
 
+# This next line basically means "it's okay for the root user to update the repo from within a cron
+# job, even if the repo is owned by a normal non-root user."
+git config --global --add safe.directory "$this_dir"
```

Comment:
Not sure how I feel about this: perhaps the root user should have its own copy of the repo instead. but it seems a little silly if running this script causes us to clone this repo itself again. What do others think?

Reply:
I prefer the method you've chosen here. Confusion will abound when two repo copies are on different versions or branches